utils.py

```diff
@@ -53,15 +54,9 @@ def send_email(to_email, subject, content, attachment=None):
                        smtp_skip_login=email_json["skip_auth"])
     # 邮箱正文
     # contents = ['您好，附件中有您的证书']
-    logger.info("send to " + to_email, 'with subject', subject)
+    logger.info("send to " + to_email + 'with subject' + subject)
```

Comment:
This line is in PR #106

Reply:
close duplicate PR or just let git deal with this

Reply:
ok